resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -232,7 +232,7 @@ function task_termini {
     #
     # This also allows us to prefer PE ruby over system ruby, so this will work
     # with PE as well
-    local rubylibdir=${rubylibdir:=$(/opt/puppetlabs/puppet/bin/ruby -rrbconfig -e "puts RbConfig::CONFIG['vendordir']")}
+    local rubylibdir=${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}
```

Comment:
I had a comment here but I deleted if after considering the implication. This is good as written.

Reply:
oh haha, I was like, why is github not showing me the comment I received an email about??